src/BuildScriptGenerator/PythonVersions.cs

```diff
@@ -13,7 +13,7 @@ public static class PythonVersions
         public const string Python37Version = "3.7.12";
         public const string Python38Version = "3.8.12";
         public const string Python39Version = "3.9.7";
-        public const string Python310Version = "3.10.4";
+        public const string Python310Version = "3.10.0";
```

Comment:
This line needs to be reverted as currently we have 3.10.4 as default version

Reply:
Thanks for catching this! 😃